src/components/BulkEditList/BulkEditListResult/BulkEditInAppPreviewModal/BulkEditInAppPreviewModal.js

```diff
@@ -53,6 +56,7 @@ const BulkEditInAppPreviewModal = ({
   const { contentUpdate } = useContentUpdate({ id: bulkOperationId });
   const { bulkOperationStart } = useBulkOperationStart();
 
+  const [isPreviewLoading, setIsLoadingPreview] = useState(false);
```

Comment:
I think we can use isLoading from useBulkOperationStart and we can remove this state then

Reply:
we can't use, we need this flag exactly when preview is loaded, `useBulkOperationStart` doesn't guarantee anything because preview generation is async process on BE side